enterprise/frontend/src/metabase-enterprise/whitelabel/index.js

```diff
@@ -211,6 +184,38 @@ if (hasPremiumFeature("whitelabel")) {
             widget: HelpLinkSettings,
             defaultValue: "metabase",
           },
+          {
+            key: "-metabase-illustration",
+            tab: "conceal-metabase",
+            // eslint-disable-next-line no-literal-metabase-strings -- Admin settings
+            display_name: t`Metabase illustrations`,
+            // eslint-disable-next-line no-literal-metabase-strings -- Admin settings
+            description: t`Customize each of the illustrations in Metabase`,
+            type: "hidden",
+          },
+          {
+            key: "show-metabot",
+            tab: "conceal-metabase",
+            display_name: (
+              <Text fw="bold" transform="none">{t`Metabot greeting`}</Text>
+            ),
+            description: null,
+            type: "boolean",
+            defaultValue: true,
+            widget: MetabotToggleWidget,
+          },
+          {
+            key: "login-page-illustration",
```

Comment:
It's easy enough to add 3 more settings once this PR is approved, so I want to make sure I get the foundation right first.

That's why this PR only contain one new setting.